.github/pull_request_template.md

```diff
@@ -1,32 +1,40 @@
 <!--
 
-Describe in detail the changes you are proposing, and the rationale.
+** Thank you for your contribution! Please read this carefully! **
 
-See the contributing guide:
-
-https://github.com/opentofu/opentofu/blob/main/CONTRIBUTING.md
+Please make sure you go through the checklist below. If your PR does not meet all requirements, please file it
+as a draft PR. Core team members will only review your PR once it meets all the requirements below (unless your
+change is something as trivial as a typo fix).
 
 -->
 
-<!--
+<!-- If your PR resolves an issue, please add it here. -->
+Resolves # 
```

Comment:
Do we want to consider making this mandatory? Making sure a PR always (or almost always) resolves an existing issue, and that this issue should have been accepted?